data_processing/face/tests/test_crop_and_resize_images.py

```diff
@@ -2,7 +2,7 @@
 import pytest
 
 from face.crop_and_resize_images import crop_and_resize_image, crop_and_resize_images
-from utils import Point, Region, Resolution
+from data_processing.utils import Point, Region, Resolution
```

Comment:
I think just `from utils import Point, Region, Resolution` should work.

Reply:
Yeah, that did work. Not sure why I had troubles with it earlier. Thanks!